Streetcode/Streetcode.XIntegrationTest/ControllerTests/Utils/BeforeAndAfterTestAtribute/Streetcode/ExtractTestUpdateStreetcode.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Streetcode.DAL.Entities.Streetcode;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace Streetcode.XIntegrationTest.ControllerTests.Utils.BeforeAndAfterTestAtribute.Streetcode
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
+    class ExtractTestUpdateStreetcode : BeforeAfterTestAttribute
+    {
+        public static StreetcodeContent StreetcodeForTest;
```

Comment:
I suggest changing type of `StreetCodeForTest` to **StreetcodeUpdateDTO** and adding **private** field of type **StreetcodeContent**. By doing this we get variable of type **StreetcodeUpdateDTO** when accessing `StreetcodeForTest` in `StreetcodeUpdateControllerTests`, so we dont need to use `CreateMoqStreetCodeDTO` method in `StreetcodeUpdateControllerTests`.
#### Example:
``` csharp
public static StreetcodeUpdateDTO StreetcodeForTest;
private static StreetcodeContent _streetcodeContent;
```
P.S Here field `_streetcodeContent` is needed to store original information from database;